modules/before-updating-web-console.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-web-console.adoc
+
+:_content-type: CONCEPT
+[id="before-updating-web-console{context}"]
+= Before updating using the web console
+
+Before updating using the web console, consider the following:
+
+* Your etcd backup is recent enough in case your update fails and you must restore your cluster to a previous state.
```

Comment:
I am not comfortable with suggesting customers restore the cluster from previous etc backup when update status is failing = true. Because failing = true is the indication that update is failing

Reply:
Do you want this list item to be removed?

Reply:
Let me know what you think. I have updated to "You have recently backed up etcd" we can completely remove though if you would prefer.